shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -381,6 +462,18 @@ exports.processStreamRecordsInteractor = async (
       },
     );
 
+    await processMessageEntries({
+      applicationContext,
+      messageRecords,
+      utils,
+    }).catch(err => {
+      applicationContext.logger.error(
+        "failed to process message records',",
+        err,
```

Comment:
can you encapsulate the error into an object and clean up the log message just a bit 🙏